clients/drcachesim/reader/compressed_file_reader.cpp

```diff
@@ -1,5 +1,5 @@
 /* **********************************************************
```

Comment:
About the PR title: this is part 3 right? edbca7bf0667e5abf2a3dcf017fd53fdaff1a8ce was part 2.

Reply:
Yes, right.